src/pages/import/components/TimetableActionButtons.tsx

```diff
@@ -0,0 +1,15 @@
+import { VStack } from '@chakra-ui/layout';
+
+import { Timetable } from 'lib/fetchers';
+
+import { ImportTimetable } from './ImportTimetable';
+import { ReplaceTimetable } from './ReplaceTimetable';
+
+export function TimetableActionButtons({ loading, data }: { loading: boolean; data: Timetable }): JSX.Element {
```

Comment:
nit: usually try to define these props with a `Props` type when there's more than 1 prop